src/components/OverflowList/OverflowList.tsx

```diff
@@ -42,62 +41,68 @@ export const OverflowList = ({ className, children, ...other }: Props) => {
   /**
    * Target overflow list and overflow list inner DOM elements
    */
-  const overflowListRef = useRef() as MutableRefObject<HTMLDivElement>;
-  const overflowListInnerRef = useRef() as MutableRefObject<HTMLUListElement>;
+  const overflowListRef = useRef<HTMLDivElement>(null);
+  const overflowListInnerRef = useRef<HTMLUListElement>(null);
   /**
    * Set right and left gradients on tables.
    */
   const setShadows = () => {
-    /**
-     * Target the actual overflow list inside overflow list
-     */
-    const overflowListItems = Array.from(overflowListInnerRef.current.children);
-    let childrenWidth = 0;
-
-    overflowListItems.forEach(function (item) {
-      const itemMarginLeft = parseInt(
-        getComputedStyle(item).marginLeft.slice(0, -2),
-      );
-      childrenWidth += item.clientWidth + itemMarginLeft;
-    });
-    if (overflowListItems) {
+    if (overflowListInnerRef.current) {
```

Comment:
This code is definitely copy and pasted.

Reply:
Could we invert this if check to avoid indenting further? Like
```
      if (!overflowListInnerRef.current) {
        return;
      }

      /**
       * Target the actual overflow list inside overflow list
       */
      const overflowListItems = Array.from(
        overflowListInnerRef.current.children,
      );
      [...]
```

Reply:
Good idea